.github/workflows/ci.yml

```diff
@@ -82,6 +83,7 @@ jobs:
     - name: Drop, Create, Migrate, and Validate DB
       env:
         ILIOS_DATABASE_URL: mysql://root:root@127.0.0.1:3306/ilios?serverVersion=5.7
+        MESSENGER_TRANSPORT_DSN: doctrine://default
```

Comment:
Is this duplication needed? I would think the definition on line 17 would do for here.

Reply:
probably not. i was trying to overcome the build error i'm seeing right now. evidently, this duplication isn't fixing it. will remove it again.

Reply:
I'm guessing you need it in https://github.com/ilios/ilios/blob/master/Dockerfile#L60 (and maybe some other lines in that file as well).

Reply:
:heavy_check_mark: